config/bootstrap.js

```diff
@@ -11,6 +11,11 @@
 
 module.exports.bootstrap = async function() {
 
+  sails.log.debug('Environment Variables:');
+  _.forIn(process.env,(value, key) => {
+    sails.log.debug(`${key} : ${value}`);
+  });
+
```

Comment:
We may not want to be logging out environment variables ... there may be some secrets in here?

Reply:
Maybe it's ok because it's only done in debug mode .... I could be convinced.

Reply:
I agree, not great to have this - removed.